crates/services/gas_price_service/src/v1/da_source_service.rs

```diff
@@ -178,10 +184,11 @@ mod tests {
 
         // given
         let l2_height = 10;
-        let recorded_height = 9;
+        let starting_recorded_height = 2;
+        let expected_recorded_height = 9;
         let unexpected_costs = DaBlockCosts {
```

Comment:
nit: I think this is probably misnamed but that would be my fault.
```suggestion
        let costs = DaBlockCosts {
```

Reply:
addressed in 53a96cfc31